gbpservice/nfp/configurator/lib/schema.py

```diff
@@ -81,24 +81,10 @@
               'stitching_cidr': "",
               'stitching_interface_index': "",
               'provider_mac': "",
-              'stitching_mac': "",
```

Comment:
Yes. Fine. But, both of this PR and the review comment addressed PR should get merged together